docs/topics/request-response.rst

```diff
@@ -56,7 +56,7 @@ Request objects
       ``body`` is not given, an empty string is stored. Regardless of the
       type of this argument, the final value stored will be a ``str`` (never
       ``unicode`` or ``None``).
-    :type body: str or unicode
+    :type body: str
```

Comment:
I think this is incorrect; docs above are also incorect. There is no `unicode` in Python 3, and request.body is bytes, not str.